src/preRelease/installer.ts

```diff
@@ -71,9 +69,7 @@ function isNewerAlphaVersionAvailable(
   const availableSemverCoerce = semver.coerce(availableVersion)?.version || "";
 
   const isNewerVersion =
-    !!currentVersion &&
-    semver.gt(availableVersion, currentVersion) &&
-    semver.neq(availableSemverCoerce, badVersion);
```

Comment:
why? not relevant?

Reply:
It's very old issue.. thought I can remove it already. I can re-add it if you think it's still needed.